packages/nimble-components/src/theme-provider/design-token-names.ts

```diff
@@ -226,13 +242,18 @@ const tokenSuffixes = [
     'FontSize',
     'TextTransform',
     'FontFamily',
+    'BoxShadow',
+    'OffsetX',
+    'OffsetY',
     'Font',
     'Size',
     'Width',
     'Height',
     'Delay',
     'Padding',
-    'Color'
+    'Color',
+    'ShadowColor',
```

Comment:
Based on the source comment on the top for this to properly recognize as a suffix it should go higher in the list.

But based on the PR comment below I think we should remove it anyway. I think only BoxShadow will ultimately be added as a new suffix.